src/SensitiveFloats.jl

```diff
@@ -161,22 +161,15 @@ function combine_sfs!{ParamType <: ParamSet,
         sf_result::SensitiveFloat{ParamType, T1},
         v::T1, g_d::Vector{T2}, g_h::Matrix{T3};
         calculate_hessian::Bool=true)
-
-    # TODO: time consuming **************
-
-    # TODO: this line is allocating a lot of memory and I don't know why.
-    # Commenting this line out attributes the same allocation to the next line.
-    # Is memory being allocated lazily or misattributed?
-    @assert g_h[1, 2] == g_h[2, 1]
```

Comment:
In case you wonder, Hessians can be constructed enforcing exact symmetry, by populating mirror elements with exactly the same float.  Everything I've written does it this way.  That's why this is ok (notice it hasn't failed yet).

Incidentally, the second TODO can be deleted, as it was due to a bug in the profiler.